nixos/modules/security/acme.nix

```diff
@@ -363,6 +376,7 @@ in
                         '';
                       in
                         "+${script}";
+                    Restart = "on-failure";
```

Comment:
I don‘t have a source right now. But I think I remember that restart on failure is considered evil by lets encrypt. So enabling this on default in nixos might actually be a ddos.
But I might be wrong on this …

Reply:
You are probably right.

I was trying to do best effort to compensate for something in Lego (that I do not understand https://github.com/go-acme/lego/issues/1192). But it is just one particular case of failure.